### ISSUE TITLE
Make the COMIT node use the Ledger Query Service for querying bitcoin blockchain information

### ISSUE THREAD

**Issue by thomaseizinger** [comit-network/comit-rs#245]
Not in scope for this ticket:

- Replacing all(!) the poking endpoints, only the ones for bitcoin should be replaced.

**Remaining work (handover from @thomaseizinger)**
- [x] Move non-decision making code from `handle` to `swap_protocol::config`
  - [x] Have the `config` create the payload directly instead of it being returned by `handle` and wrapped in SwapResponse
- [x] Create the query/future in `swap_protocol::config` after item above

Depends on #244.
